TACC-Cloud/geoapi
# Task/DES-2001: Project links as observables (subscribe to project-user data).

## Overview: ##

## PR Status: ##


## Related Jira tickets: ##

* [DES-2001](https://jira.tacc.utexas.edu/browse/DES-2001)

## Summary of Changes: ##
When saving/linking the map to a project, ensure that an observable project is also created.

## Testing Steps: ##
1. Save a map to a project and ensure that users are synced. (test with TACC-cloud/hazmapper#67)

## Notes: ##
TODO: 
1. Fix tests for new routes
2. Clean up logic more
